suppressions.txt

```diff
@@ -1,4 +1,3 @@
 useStlAlgorithm
 unusedFunction
-toomanyconfigs
```

Comment:
Why?

Reply:
produced cpp-check error. might be due to missing psa_driver.c
https://github.com/aosedge/aos_core_lib_cpp/actions/runs/10739968945/job/29786976717
replaced by suppressing psa-driver dir entirely as it produces other warnings: